tests/unit/drivers/test_cache_driver.py

```diff
@@ -26,11 +28,11 @@ def docs(self):
         return DocumentSet(list(random_docs(10)))
 
 
-def test_cache_driver_twice(tmp_path):
-    filename = tmp_path / 'test-tmp.bin'
+def test_cache_driver_twice(tmpdir):
     docs = DocumentSet(list(random_docs(10)))
     driver = MockCacheDriver()
-    with DocIDCache(filename) as executor:
+    # FIXME DocIdCache doesn't use tmpdir, it saves in curdir
```

Comment:
you need to use `test_metas`

Reply:
Tried, same problem

Reply:
Is not that u do `DocIDCache(metas=test_metas)` , metas can only be passed by `BaseExecutor.load_config` I think.

What you can do is to set `executor.workspace = tmpdir`

Reply:
But here it works https://github.com/jina-ai/jina/blob/f143a073857bd547e4af12caf127e526217d40d6/tests/unit/executors/indexers/test_numpyindexer.py#L23 What am I missing?